src/components/TableComponent/TableComponent.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import './TableComponent.scss';
+import { TableRow } from './index';
+import { SearchTerms } from '../../types/types';
```

Comment:
Doesn't seem to be used. Also these types of errors can be easily caught by adding a linter(this should be done in a separate PR). The workflow can probably be copied from here: https://github.com/RegionX-Labs/CoretimeHub/blob/master/.github/workflows/ci.yml.